src/Auth/index.ts

```diff
@@ -4,6 +4,8 @@ import { RegisterAddressRsp } from "./types";
 
 export default class Auth {
   private keyService: KeyService;
+  private getTokenUrl: string = 'https://account.mytiki.com/api/latest/auth/token';
```

Comment:
Change to `tokenUrl`. "Get" should only be used in methods. Properties names should start with nouns, not verbs.